CHANGELOG.md

```diff
@@ -23,7 +23,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Implement `AddAssignUnchecked` and `SubAssignUnchecked` for `StorageUint`. #418
 - Implement `MethodError` for `safe_erc20::Error`. #402
 - Use `function_selector!` to calculate transfer type selector in `Erc1155`. #417
-- Update internal functions of `Erc721` and `Erc721Consecutive` accept reference to `Bytes`. #437
+
+### Changed (Breaking)
+
+- Update internal functions of `Erc721` and `Erc721Consecutive` to accept a reference to `Bytes`. #437
+- Stop supporting reentrancy, and borrow `self` immutably in `IErc721::_check_on_erc721_received` and `IErc1155::_check_on_erc1155_received`. #440
```

Comment:
Update following to the change suggested by `clippy`.

Reply:
Applied changes suggested by `clippy` and updated `CHANGELOG` ✅